### PR TITLE
Title bug fixes & minor improvements

### PR DESCRIPTION
**Pull request by haxvzje**
+ Fixed series.id
+ Remove unnecessary characters in volume.title
+ Added countdown timer for rate limit
- The default value of "specvol" flag changes to TRUE

image

image

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -115,7 +116,8 @@ func main() {
 
 	doc.Find(".volume-list").Each(func(_ int, selection *goquery.Selection) {
 		var vol Volume
-		vol.Title = strings.TrimSpace(selection.Find(".sect-title").Text())
```
Comment (Kinue72) [inline main.go:118]: You should compile regex to a global variable

File: main.go [hunk 2]
```diff
@@ -115,7 +116,8 @@ func main() {
...
+		re := regexp.MustCompile(`[\*\@\$\%\^\(\)]`)
```
Comment (Kinue72) [inline main.go]: You should compile regex to a global variable

Comment (haxvzje) [reply main.go]: solved!

**Comment by Kinue72** [comment]
lgtm

**Pull request merged**